src/spyglass/utils/nwb_helper_fn.py

```diff
@@ -96,6 +96,11 @@ def get_nwb_file(nwb_file_path):
         nwbfile = io.read()
         __open_nwb_files[nwb_file_path] = (io, nwbfile)
 
+    __open_nwb_files.clear()
```

Comment:
Will this clear the cache every time?

Reply:
Yes. I agree this doesn't seem ideal and have removed the line.